dune-project

```diff
@@ -1,3 +1,2 @@
-(lang dune 1.4)
-
+(lang dune 1.6)
```

Comment:
There are a few place in the doc that needs updating to reflect this: `Makefile.dune` and `dev/doc/build-system.dune.md`.